### DISCUSSION TITLE
Progress on Serwist Integration

### DISCUSSION THREAD

**Discussion by n-ce** [vite-pwa/vite-plugin-pwa#765] [General] [answered]
How is the progress going on on replacing workbox with serwist? I currently see no code being part of it. Is there still intention to do this?

**Comment by userquin** [comment, marked as answer]
serwist removed the `generateSW` module (blocks the migration), check also this comment https://github.com/serwist/serwist/issues/19#issuecomment-1872182533

**Reply by n-ce** [reply]
It seems to be diverging from the interests of vite-pwa, so are we stuck with workbox now?

**Reply by userquin** [reply]
Maybe we can fork workbox similar way serwist did, I'll ask Thomas Steiner about the new Google team in the workbox repo...
